plugins/stored-constraints/src/utils.ts

```
@@ -0,0 +1,61 @@
+  record: FlatfileRecord,
+  field: Flatfile.Property,
+  deps: any,
+  sheet: Flatfile.SheetResponse
+) => {
+  const storedConstraint = getStoredConstraints(field.constraints || []).find(
+    (fieldConstraint) => fieldConstraint.validator === constraint.validator
+  )
+  const { config = {} } = storedConstraint || {}
+  const constraintFn = constraint.function.startsWith('function')
+    ? constraint.function.includes('function constraint')
+      ? eval(
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)